specs/engage-digital_openapi3.yaml

```diff
@@ -8483,6 +8487,78 @@ components:
           items:
             type: string
           type: array
+        imap_smtp_email:
+          description: Email address to synchronize. Only on Email sources
+          type: string
+        imap_mailboxes_to_import:
+          description: Folders to synchronize. Only on Email sources
+          items:
+            type: string
+          type: array
+        imap_address:
+          description: IMAP server address. Only on Email sources
+          type: string
+        imap_port:
+          description: IMAP server port. Only on Email sources
+          format: int32
+          type: integer
+        imap_username:
+          description: IMAP username. Only on Email sources
+          type: string
+        imap_use_ssl_tls:
+          description: Use SSL/TLS protocol to establish secure IMAP connection. Only on Email sources
+          type: boolean
+        imap_use_start_tls:
+          description: Use STARTTLS protocol to establish secure IMAP connection. Only on Email sources
+          type: boolean
+        imap_authentication_type:
+          description: IMAP Authentication type ("basic" or "microsoft_identity_platform"). Only on Email sources
+          type: string
+        imap_ssl_verify_mode:
+          description: IMAP SSL verification mode. ("NONE", "PEER", "CLIENT_ONCE" or "FAIL_IF_NO_PEER_CERT"). Only on Email sources
+          type: string
+        smtp_address:
+          description: SMTP server address. Only on Email sources
+          type: string
+        smtp_port:
+          description: SMTP server port. Only on Email sources
+          format: int32
+          type: integer
+        smtp_username:
+          description: SMTP username. Only on Email sources
+          type: string
+        smtp_use_ssl_tls:
+          description: Use SSL/TLS protocol to establish secure SMTP connection. Only on Email sources
+          type: boolean
+        smtp_use_start_tls:
+          description: Use STARTTLS protocol to establish secure SMTP connection. Only on Email sources
+          type: boolean
+        smtp_authentication_type:
+          description: SMTP Authentication type ("basic" or "microsoft_identity_platform"). Only on Email sources
+          type: string
+        smtp_ssl_verify_mode:
+          description: SMTP SSL verification mode ("NONE", "PEER", "CLIENT_ONCE" or "FAIL_IF_NO_PEER_CERT"). Only on Email sources
+          type: string
+        smtp_helo_domain:
+          description: Used to identify HELO/EHLO when connecting to the SMTP server. Only on Email sources

```

Comment:
What is the helo domain ? I'm not sure to understand it

Reply:
absolutely no idea, Yoann should know